ocs_ci/deployment/deployment.py

```diff
@@ -114,6 +114,7 @@ def deploy_cluster(self, log_cli_level='DEBUG'):
                     self.deploy_ocp(log_cli_level)
                     self.post_ocp_deploy()
                 except Exception as e:
+                    config.ENV_DATA['is_ocp_deployment_failed'] = True
```

Comment:
What about to use RUN section for that as it's more relevant to run?

Reply:
Done